.gitignore

```diff
@@ -117,4 +117,7 @@ dist
 .yarn/unplugged
 .yarn/build-state.yml
 .yarn/install-state.gz
-.pnp.*
\ No newline at end of file
+.pnp.*
+
+# Useless macOS files
```

Comment:
I feel like saying `useless` is a _bit_ harsh 😆 

https://en.wikipedia.org/wiki/.DS_Store